pyproject.toml

```diff
@@ -6,18 +6,18 @@ authors = []
 
 [tool.poetry.dependencies]
 atoti = {extras = ["sql"], version = "*"}
-pandas = "1.3.3"
-python = "3.9.7"
-# poetry deletes this package when the --no-dev flag is enabled because it is a dependency of requests, however it is needed for package installation.
-certifi = "^2021.10.8"
+# Poetry deletes this package when the --no-dev flag is enabled because it is a dependency of requests, however it is needed for package installation.
+certifi = "*"
+pandas = "*"
```

Comment:
Using `*` won't prevent Dependabot from submitting PRs to update deps. Proof: #21.